imports/ui/admin/Admin.html

```diff
@@ -41,6 +41,27 @@
                 <input required id="japaneseName" name="japaneseName" class="form-control" placeholder="Japanese League Name...">
             </div>
         </div>
-        <button type="submit" style="width: 25%" class="btn btn-primary">Add</button>
+        <button type="submit" class="btn btn-primary">Add Player</button>
     </form>
+    <br>
+    <div class="names">
+        <table style="width: 50%" class="table">
+            <thead>
+                <tr>
+                    <th>Name</th>
+                    <th>Japanese</th>
```

Comment:
Japanese and Hong Kong are not really descriptive enough labels.

Reply:
The person using the page should understand what they mean, especially since the first column is "Name".

What would they get confused about in this situation?

Reply:
Just seems like bad practice.